### ISSUE TITLE
Handwrite

### ISSUE THREAD

**Issue by Pantyhose-X** [florisboard/florisboard#2443]
### Feature idea

I don't know how to type. Traditional Chinese, Japanese, Korean. Handwrite does a great job of helping me type what I need.

**Comment by Zig-Rust-Odin** [comment]
chinese Japanese handwriting engine
https://kiang.org/jordan/software/hanzilookup/
https://github.com/tegaki/tegaki
https://github.com/gugray/HanziLookupJS
https://gitee.com/hebinq/shuoxie
https://github.com/quen/kanjirecog/tree/android
https://github.com/Saren-Arterius/google-chinese-handwriting-ime
https://popolon.org/gblog3/?p=447&lang=en

**Comment by My1** [comment]
even handwriting for Latin-based languages would be pretty neat to have right inside for pen-devices so you can use the same keyboard for typing and handwriting

**Comment by My1** [comment]
although isnt this more or less a dupe of #233 which also proposes handwriting?

**Comment by mikeloeven** [comment]
Yeah an issue several years old with no real traction. But honestly right now for handwriting the only real option is Gboard which you just know is going to be training off you because its a google app. I have been searching for hours and cant find a good handwriting input for android that is FOSS

**Comment by tsiflimagas** [comment]
While I understand that some of you are eager to have this nice feature, as the author of #233 also mentions, it is a hard to implement one, and apart from that, there's a long queue of features waiting ahead of this one. I hope it will eventually arrive, to your satisfaction. In the meanwhile, let's keep the discussion at #233.
